projects/igniteui-angular/src/lib/navigation-drawer/navigation-drawer.component.ts

```diff
@@ -612,18 +614,26 @@ export class IgxNavigationDrawerComponent implements
     }
 
     private getWindowWidth() {
+        if (!this.platformUtil.isPlatformBrowser()) {
```

Comment:
This can likely stay (though I'd prefer a return value of something at least (0)).
Frankly, this only affects `checkPinThreshold` and while the current logic there will is fin like that, the whole check is pointless if there's no window to measure so it should be prevented instead.

Reply:
Actually in PWA with app shell, while building the app `ngAfterContentInit` is called. This calls `checkPinThreshold`. I am moving now the check to `checkPinThreshold` as it seams to be the only place where `getWindowWidth` is called with no browser.